File: repos_estudiantes_ALVAREZ_primera_entrega/ARUQUIPA_VARGAS_ANA_GLORIA/INTRODUCCION/Ejercicio5Py_java/main.py
from Persona import Persona

def main():
    persona1 = Persona("Juan", "Pérez", "Gómez", 20, "1234567")
    persona2 = Persona("María", "Pérez", "López", 17, "7654321")

    print("Datos de persona 1:")
    persona1.mostrarDatos()

    print("\nDatos de persona 2:")
    persona2.mostrarDatos()

    print("\nVerificación de mayoría de edad:")
    persona1.mayorDeEdad()
    persona2.mayorDeEdad()

    print("\nModificando edad de persona 2:")
    persona2.modificarEdad(19)
    persona2.mostrarDatos()

    if persona1.getApellidoPaterno().lower() == persona2.getApellidoPaterno().lower():
        print("Ambas personas tienen el mismo apellido paterno.")
    else:
        print("Las personas tienen apellidos paternos diferentes.")

if __name__ == "__main__":
    main()

File: repos_estudiantes_ALVAREZ_primera_entrega/ARUQUIPA_VARGAS_ANA_GLORIA/INTRODUCCION/Ejercicio5Py_java/Main.java
package Ejercicio5Py_java;

public class Main {
    public static void main(String[] args) {
        Persona persona1 = new Persona("Juan", "Pérez", "Gómez", 20, "1234567");
        Persona persona2 = new Persona("María", "Pérez", "López", 17, "7654321");
        System.out.println("---- Persona 1 ----");
        persona1.mostrarDatos();
        System.out.println("Mayor de edad: " + persona1.mayorDeEdad());
        System.out.println("\n---- Persona 2 ----");
        persona2.mostrarDatos();
        System.out.println("Mayor de edad: " + persona2.mayorDeEdad());
        persona2.modificarEdad(19);
        System.out.println("\nEdad modificada de Persona 2:");
        persona2.mostrarDatos();
        if (persona1.getPaterno().equalsIgnoreCase(persona2.getPaterno())) {
            System.out.println("\nAmbas personas tienen el mismo apellido paterno.");
        } else {
            System.out.println("\nLas personas tienen apellidos paternos diferentes.");
        }
    }
}

